packages/demo-wallet-shared/methods/sep31Send/pollTransactionUntilReady.ts

```diff
@@ -32,14 +33,16 @@ export const pollTransactionUntilReady = async ({
     }
 
     // eslint-disable-next-line no-await-in-loop
-    const resultJson = await result.json();
+    resultJson = await result.json();
     log.response({
       title: `GET \`/transactions/${transactionId}\``,
       body: resultJson,
     });
-    transactionStatus = resultJson.transaction.status;
+    transactionStatus = resultJson.transaction?.status;
```

Comment:
Do we want to set a default status here or no status is acceptable?

Reply:
There's no need fr a default value because all we want in this part of the code is to verify if the status is `TransactionStatus.PENDING_SENDER`. Here's the full code snippet for reference:
https://github.com/stellar/stellar-demo-wallet/blob/9c7e3f3f99a6563df26b95a385c883a300356769/packages/demo-wallet-shared/methods/sep31Send/pollTransactionUntilReady.ts#L19-L44